be/src/storage/persistent_index.h

```diff
@@ -625,6 +632,9 @@ class PersistentIndex {
 
     bool is_error() { return _error; }
 
+    void set_dump_snapshot(bool dump_snapshot) { _dump_snapshot = dump_snapshot; }
+    void set_flushed(bool flushed) { _flushed = flushed; }
```

Comment:
Why add this two function? It seams useless?

Reply:
Just change the two functions private to protect in persistentIndex, in order to it can be used in subclass lakePersistentIndex

Reply:
But it seams that `set_dump_snapshot` isn't used in `lakePersistentIndex` code? Do I miss something?

Reply:
ok